C/plugins/north/OMF/include/omf.h

```diff
@@ -101,6 +101,20 @@ class OMF
 		// Destructor
 		~OMF();
 
+		void		setOMFVersion(std::string& omfversion)
```

Comment:
Not a new problem but there is a lot of extra space and deep indentation throughout the code. If you use Visual Studio Code, it has a feature for fixing this. Right click on the document and choose _Format Document_.

Reply:
Hell will freeze over before I use Visual studio!